lrpar/doc/src/quickstart.md

```diff
@@ -0,0 +1,94 @@
+# Quickstart Guide
+
+This is a short guide to get you started. For a more detailed explanation about
+what's going on, please refer to the [tutorial](tutorial/index.html).
+
+We begin by creating a `Cargo.toml` file and load the necessary dependencies:
+
+```toml
+[package]
+name = "project"
+version = "0.0.1"
+authors = ["<authors>"]
+
+[[bin]]
+doc = false
+name = "project"
+
+[build-dependencies]
+lrlex = { path="<path to lrlex>" }
+lrpar = { path="<path to lrpar>" }
+
+[dependencies]
+cfgrammar = { path="<path to cfgrammar>" }
+lrlex = { path="<path to lrlex>" }
+lrpar = { path="<path to lrpar>" }
+```
+
+In the root of the project we create a file `build.rs`, which processes our lexer and grammar files.
+Here we can also choose the integer size for the token ID's and whether we want to generate a default
```

Comment:
"'" is a possessive in this context which doesn't make sense: "IDs" is all we need.